docs/contribution/documentation_cheatsheet.md

```diff
@@ -18,35 +48,51 @@ A link to {class}`Acts::Volume`.
 
 * Code: 
 
-````markdown
-```{doxygenclass} Acts::Volume
+```text
+:::{doxygenclass} Acts::Volume
 ---
 members: center
 ---
+:::
 ```
-````
 
 * Result:
 
-```{doxygenclass} Acts::Volume
+:::{doxygenclass} Acts::Volume
 ---
 members: center
 ---
-```
+:::
 
 (cheatsheetlabels)=
 ## Cross referencing and labels
 
 * Setting a label:
 
-```markdown
+```text
 (cheatsheetlabels)=
 ```
 
 * Referencing a label:
 
-```markdown
+```text
 Click [here](cheatsheetlabels) to come to the label.
```

Comment:
You didn't change this, but maybe you could mention that you can leave the `[]` blank and it will grab the headline text of the target automatically.